src/components/fargekategori/FargekategoriFeilhandtering.tsx

```diff
@@ -12,20 +12,24 @@ function isString(data: string | {data: string[]; errors: string[]}): data is st
 }
 
 const mapErrorToText = (okFnrs: string[], errorFnrs: string[], status: number | undefined) => {
-    if (!okFnrs?.length) {
+    const ingenGikkBra = !okFnrs?.length;
+    const bareEnFeilet = errorFnrs.length == 1;
```

Comment:
Lokalt så kræsjer det uten et spørsmålstegn her når man blokkerer kallet til `/fargekategorier`
```suggestion
    const bareEnFeilet = errorFnrs?.length == 1;
```